curriculum/challenges/english/25-front-end-development/lecture-importance-of-semantic-html/6729959bf9c8e835f46b3f78.md

```diff
@@ -10,6 +10,36 @@ dashedName: when-should-you-use-the-emphasis-element-over-the-idiomatic-text-ele
 
 Watch the lecture video and answer the questions below.
 
+# --transcript--
+
+When should you use the emphasis element over the idiomatic text element?
+
+These elements are very closely related to the concepts of presentational and semantic HTML. The idiomatic text element, `i`, was originally used for presentational purposes to display the text in italics. But now, it is frequently used for highlighting alternative voice or mood, idiomatic terms from another language, technical terms, and thoughts.
+
+Here is example from the official HTML spec, using the `i` element to show an idiomatic phrase in French:
+
+```html
+<p>There is a certain <i lang="fr">je ne sais quoi</i> in the air.</p>
+```
+
+The `lang` attribute inside the open `<i>` tag is used to specify the language of the content. In this case, the language would be French. The `i` element does not indicate if the text is important or not, it only shows that it's somehow different from the surrounding text.
+
+If you do need to emphasize the importance of the text, you can use a similar semantic element called the emphasis element, `em`. This is usually recommended if you need to provide more context. You should use this element for parts of the text that require a special emphasis compared to surrounding text. It's usually limited to only a few words, because it can alter the meaning of the sentence.
+
+This is an example of the emphasis element element within a paragraph:
+
+```html
+<p>
+  Never give up on <em>your</em> dreams.
+</p>
+```
+
+You can see the sentence "Never give up on your dreams". Notice that the word "your" will be emphasized, because it's inside this element. In the browser you would see the word "your" is italicized to tell readers that this is an important word in the sentence.
```

Comment:
```suggestion
You can see the sentence `Never give up on your dreams`. Notice that the word `your` will be emphasized, because it's inside this element. In the browser you would see the word `your` is italicized to tell readers that this is an important word in the sentence.
```